transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebAcceptProcessor.java

```diff
@@ -100,7 +100,7 @@ protected void flushInternal(final WsebSession session) {
         }
 
         // get write request queue and process it
-        final WriteRequestQueue writeRequestQueue = session.getWriteRequestQueue();
+        final WriteRequestQueue writeRequestQueue = session.getTransportSession().getWriteRequestQueue();
```

Comment:
Basically it is the processor for TransportSession now, but I was not able to directly set it as the processor on TransportSession because of type checking (would require extensive code changes which I don't want to bite off), that's why I am using TransportProcessor instead, which just wraps the WsebAcceptProcessor or WsebConnectProcessor instance.